transact/alchemy_simulateExecution.yaml

```diff
@@ -647,3 +649,11 @@ components:
             - type: string
         indexed:
           type: boolean
+
+    revertMsg:
+      type: [string, 'null']
```

Comment:
any reason we use the [] syntax instead of

```
type:
 - string
 - null
```

lets chose one but be consistent across specs :)

Reply:
Just searched in the codebase and currently we don't have any other multi-type variables, this is the only instance I could find. So let's keep `[]` as I think it's easier to read the multiple types of one variable in an array. 

But can switch to the other one if you want.

Reply:
we actually do it for a lot of fields already
some type fields and all required fields

https://github.com/alchemyplatform/docs-openapi-specs/blob/main/transact/alchemy_simulateAssetChanges.yaml#L64

i dont mind what we use but need to make sure were consistent across all specs
granted we use a mix of both in this file already but we should fix